src/main/java/de/uniba/dsg/serverless/cli/performance/GooglePerformanceDataUtility.java

```diff
@@ -0,0 +1,66 @@
+package de.uniba.dsg.serverless.cli.performance;
+
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import de.uniba.dsg.serverless.cli.CustomUtility;
+import de.uniba.dsg.serverless.model.SeMoDeException;
+import de.uniba.dsg.serverless.model.WritableEvent;
+import de.uniba.dsg.serverless.provider.aws.AWSLogHandler;
+import de.uniba.dsg.serverless.provider.google.GoogleLogHandler;
+import de.uniba.dsg.serverless.util.BenchmarkingRESTAnalyzer;
+
+public class GooglePerformanceDataUtility extends CustomUtility implements PerformanceData {
+	
+	private static final Logger logger = LogManager.getLogger(GooglePerformanceDataUtility.class.getName());
+
+	public GooglePerformanceDataUtility(String name) {
+		super(name);
+	}
+
+	@Override
+	public void start(List<String> args) {
+		
+		if (args.size() < 3) {
+			logger.fatal("Wrong parameter size: \n(1) Function Name"
+					+ "\n(2) Start time filter of performance data" + "\n(3) End time filter of performance data"
+					+ "\n(4) Optional - REST calls file");
+			return;
+		}
+		
+		String functionName = args.get(0);
+		String startTimeString = args.get(1);
+		String endTimeString = args.get(2);
+		
+		try {
+			
+			this.validateStartEnd(startTimeString, endTimeString);
```

Comment:
Validation is removed in AWSPerformanceData but still used in AzurePerformanceData and GooglePerformanceData. They should all do the same.
See other comment on PR #20

Reply:
Integrated in #e1137bd